doc/how_to_use.md

```diff
@@ -142,3 +142,48 @@ Furthermore, we can enable it, so that it is activated by default on future boot
 
 [un]: https://github.com/sozu-proxy/sozu/blob/master/os-build/systemd/sozu.service.in
 [gen]: https://github.com/sozu-proxy/sozu/blob/master/os-build/exherbo/generate.sh
+
+## PROXY Protocol
+
+Sōzu use its own IP stack to get connected on remote servers. Because of this, we lose the initial L3/4 client connection information (like source and destination IP and port).
```

Comment:
why do you say it has its own IP stack? it uses mio's sockets, which use POSIX sockets provided by the OS

Reply:
I'll modify that for "IP infos".